### ISSUE TITLE
large GPT-2 model has been replicated

### ISSUE THREAD

**Issue by nestordemeure** [minimaxir/gpt-2-simple#111]
I just saw that the large GPT-2 model has been replicated, you might be able to integrate it into your library (if it is not too large for consumer grade RAM) : [OpenGPT-2: We Replicated GPT-2 Because You Can Too](https://medium.com/@vanya_cohen/opengpt-2-we-replicated-gpt-2-because-you-can-too-45e34e6d36dc)

**Comment by johndpope** [comment]
https://drive.google.com/drive/u/0/folders/1KfK5MXtvgH8C615UUZoKPIUVJYIdJxX1

**Comment by minimaxir** [comment]
The replicated GPT-2 might not be a drop-in replacement in terms of architecture. Would have to test to make sure if it's a drop-in replacement.

**Comment by johndpope** [comment]
this hparmas.json maybe a starting point for config file https://raw.githubusercontent.com/ConnorJL/GPT2/master/1.5B.json

**Comment by GenTxt** [comment]
It's a gpt2 version of the "grover mega" model. The original is not currently available for public download.  Parameter file here works with model and the grover repo:

https://github.com/rowanz/grover/tree/master/lm/configs - see mega.json

Can load and generate sample articles using only an 8 gig 1070. The .jsonl format used by grover for input and output is, well, "ugly" to say the least (see repo example.) If someone can modify to fine-tune on standard text files and output same as per other repos then it might prove useful.

This custom 1.5B model bodes well (for generation at least) if OpenAi releases their 1.5B. Fine-tuning with adam will be another issue as it currently is with 774M.

On a side note: Have sgd/ all layers/ memory saving gradients=True working with nshepperd repo and 774M. Results ok but not as good as fine-tuning 345M adam. Have not had similar success with gpt2simple.

Looking forward to future code upgrades and solutions.

**Comment by woctezuma** [comment]
> The cost of training the model from scratch using our code is about $50k. 

Wow!
